Dockerfile.slim

```diff
@@ -4,6 +4,7 @@ LABEL maintainer="sig-platform@spinnaker.io"
 ENV KUSTOMIZE_VERSION=3.8.6
 ENV KUSTOMIZE4_VERSION=4.5.5
 ENV PACKER_VERSION=1.8.1
+ENV HELMFILE_VERSION=0.153.1
```

Comment:
I noticed that [0.154.0](https://github.com/helmfile/helmfile/releases/tag/v0.154.0) has now appeared.  I'm OK to go ahead with 0.153.1 and update in a follow-up PR.

Reply:
Since this PR, 0.155.0 has come out. Once this will be merged we are going to bump the version of helm file.